config-examples/users.yaml

```diff
@@ -0,0 +1,24 @@
+---
+roles:
+  admin:
+    read: true
+    write: true
+    read_rest: true
+    write_rest: true
+  reader:
+    read: true
+    read_rest: true
+  writer:
+    write: true
+    write_rest: true
+
+users:
+  - username: admin_user
+    password: admin_pass
```

Comment:
Will password be stored as plain text?

Reply:
Hm, I've implemented simplest storage and it do just 2 things:
1. parses config
2. allows to get user by username with his password [as it was written in config] and perms

I think, we should store hashed password (in config) and set specific hash function, which will resolve correct passwords to that hashes, but I also think, that it may be weird to implement it on this storage level.
This config is just an example of what can be parsed with this storage.